docs/platforms/javascript/common/tracing/configure-sampling/index.mdx

```diff
@@ -96,98 +119,16 @@ tracesSampler: (samplingContext) => {
-
-## Inheritance in Distributed Tracing
-
-In distributed systems, trace information is propagated between services. The inheritOrSampleWith function simplifies handling parent sampling decisions:
-
-```javascript
-tracesSampler: (samplingContext) => {
-  const { name, inheritOrSampleWith } = samplingContext;
-  
-  // Apply specific rules first
-  if (name.includes('critical-path')) {
-    return 1.0; // Always sample
-  }
-  
-  // Otherwise inherit parent sampling decision or fall back to 0.1
-  return inheritOrSampleWith(0.1);
-}
-```
-This approach ensures consistent sampling decisions across your entire distributed trace. All transactions in a given trace will share the same sampling decision, preventing broken or incomplete traces.
-
-**Note:** The `inheritOrSampleWith` helper was introduced in version 9 of the SDK. For earlier versions, you can implement similar logic manually using the `parentSampled` property.
-
 ## Sampling Decision Precedence
 
 When multiple sampling mechanisms could apply, Sentry follows this order of precedence:
 
-- If `tracesSampler` is defined, its decision is used (can consider parent sampling)
-- If no `tracesSampler` but parent sampling is available, parent decision is used
+- If a sampling decision is passed to `startSpan`, that decision is used
+- If `tracesSampler` is defined, its decision is used. Although the `tracesSampler` can override the parent sampling decision, most users will want to ensure their `tracesSampler` respects the parent sampling decision. 
+- If no `tracesSampler` is defined, but there is a parent sampling decision from an incoming distributed trace, we use the parent sampling decision
 - If neither of the above, `tracesSampleRate` is used
-- If none of the above are set, no transactions are sampled (0%)
-
-## How Sampling Propagates in Distributed Traces
-
-Sentry uses a "head-based" sampling approach:
-
-- A sampling decision is made in the originating service (the "head")
-- This decision is propagated to all downstream services via HTTP headers
-
-The two key headers are:
-- `sentry-trace`: Contains trace ID, span ID, and sampling decision
-- `baggage`: Contains additional trace metadata including sample rate
-
-Sentry automatically attaches these headers to outgoing HTTP requests when using the `browserTracingIntegration`. For other communication channels like WebSockets, you can manually propagate trace information:
-
-```javascript
-// Extract trace data from the current scope
-const traceData = Sentry.getTraceData();
-const sentryTraceHeader = traceData["sentry-trace"];
-const sentryBaggageHeader = traceData["baggage"];
-
-// Add to your custom request (example using WebSocket)
-webSocket.send(JSON.stringify({
-  message: "Your data here",
-  metadata: {
-    sentryTrace: sentryTraceHeader,
-    baggage: sentryBaggageHeader
-  }
-}));
-```
+- If none of the above are set, no transactions are sampled. This is equivalent to setting `tracesSampleRate` to `0.0`.
```

Comment:
hmm this is kind of true but misleading - it is _not_ the same to omit this setting vs. setting it to `0`. The former will completely disable tracing, not propagating any tracing decision. The latter will ensure no spans are created but also that no downstream spans will be sampled because they will inherit a "no" sampling decision.

Reply:
I adjusted this to add that additional clarity! Thank you!